src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Consider renaming the enum to 'Color' instead of 'Colors' to follow naming conventions for singular names.

Reply:
done